src/lavinmq/shovel/shovel_store.cr

```diff
@@ -8,6 +8,13 @@ module LavinMQ
 
     forward_missing_to @shovels
 
+    def parse_uris(src_uri) : Array(URI)
```

Comment:
I guess the argument is `JSON::Any`? IMO we can be explicit about since `as_s` and such are JSON stuff. At first I thought the type was `String | Array(String)`.

```suggestion
    def parse_uris(src_uri : JSON::Any) : Array(URI)
```